packages/wonder-blocks-dropdown/components/__tests__/single-select.test.js

```diff
@@ -1,12 +1,16 @@
 //@flow
 import React from "react";
+import IconButton from "@khanacademy/wonder-blocks-icon-button";
 import {mount, unmountAll} from "../../../../utils/testing/mount.js";
 
 import DropdownOpener from "../dropdown-opener.js";
 import SelectOpener from "../select-opener.js";
 import OptionItem from "../option-item.js";
 import SingleSelect from "../single-select.js";
 import {keyCodes} from "../../util/constants.js";
+import SearchTextInput from "../search-text-input.js";
+
+jest.useFakeTimers();
```

Comment:
You should probably only call this in a `beforeEach()` since if another test calls `jest.useRealTimers()` then this wouldn't do anything.